_episodes/07-Practical Considerations:Reporting-results.md

```diff
@@ -23,6 +23,9 @@ As with any other statistical analysis you can expect to report various metrics
     - Precision
     - Recall
     - F1 score
+    - Mean Squared Error (MSE)
+    - Root Mean Squared Error (RMSE)
+    - Mean Absolute Error (MAE)
```

Comment:
that's a good point I think regression should probably have its own section

Reply:
👍